Sources/Recycling/ListViews/Wishlist/Demo/WishListViewDemoView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+public class WishListDataSource: NSObject {
+    let wishlist: [WishList] = WishListFactory.create()
+}
+
+public class WishListViewDemoView: UIView {
+    private lazy var dataSource: WishListDataSource = {
+       return WishListDataSource()
+    }()
+    
+    // MARK: - Setup
+    
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setupView()
+    }
+    
+    public required init?(coder aDecoder: NSCoder) { fatalError() }
+    
+    private func setupView() {
+        let view = WishListView(delegate: self, dataSource: self)
+        addSubview(view)
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Minor nitpick but I usually place this one line above and before adding it to the subview, it just feels like you don't want to add a subview before disabling the `translatesAutoresizingMaskIntoConstraints`.

Reply:
Brain fart..